src/hooks/chats/chatSidebar.ts

```diff
@@ -23,7 +23,7 @@ export const useChatSidebarControls = () => {
   const dispatch = useDispatch();
   const isExpanded = useSelector(chatVisibilitySelector);
   const chatSettings = useSelector(selectedChatSettingsSelector);
-  const newPrivateMessageRecived = useNumberOfUnreadChats();
+  const newPrivateMessageReceived = useNumberOfUnreadChats();
```

Comment:
👍 good catch. Btw, I think there is an eslint rule/extension that tracks grammar errors in an automated manner

Reply:
Ooh, that sounds tempting:) I'll update my slack comment with suggestions